### ISSUE TITLE
Fixing Package Discovery Issues with pip install -e . in Healnet

### ISSUE THREAD

**Issue by rzuberi** [konst-int-i/healnet#20]
### Issue:
The pip install -e . command failed due to multiple top-level directories (data, config, assets, etc.), causing setuptools to throw an error about ambiguous package discovery.

### Solution:
Edited setup.cfg to explicitly define package discovery:

```
[metadata]
name = healnet
version = 0.0.1
description = Scalable multi-modal deep learning architecture in PyTorch
long_description = file: README.md
author = Konstantin Hemker
author_email = konstantin.hemker@cl.cam.ac.uk
license = Apache 2.0
classifiers =
    Programming Language :: Python :: 3

[options]
packages = find:
install_requires =
    einops
    torch>=1.13.1
    torchvision>=0.14.1
    openslide-python
    seaborn
    pandas>=1.5.3
    tqdm
    python-box
    scikit-learn
python_requires = >=3.6

[options.packages.find]
include =
    healnet
    healnet.*
```
Ran the command again:

`pip install -e .`

### Result:
The package was successfully installed.

**Comment by konst-int-i** [comment]
Hi @rzuberi, 

Thanks a lot for pointing this out! I am currently working on a camera-ready release which should come in the next few days. I will make sure that this issue is addressed in the release. 

Let me know if you have any questions in the meantime. 

Thanks!

**Comment by konst-int-i** [comment]
Hi @rzuberi , 

Thanks again for pointing out the package discovery issue - this has been fixed in #22. 

I will close the issue for now, but let me know if you encounter other issues.
